immersion/kubeflow_pipelines/cicd/solutions/cloudbuild_vertex.yaml

```diff
@@ -0,0 +1,48 @@
+# Copyright 2021 Google LLC
+
+# Licensed under the Apache License, Version 2.0 (the "License"); you may not use this
+# file except in compliance with the License. You may obtain a copy of the License at
+
+# https://www.apache.org/licenses/LICENSE-2.0
+    
+# Unless required by applicable law or agreed to in writing, software 
+# distributed under the License is distributed on an "AS IS"
+# BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either 
+# express or implied. See the License for the specific language governing 
+# permissions and limitations under the License.
+
+steps:
+# Build the trainer image
+- name: 'gcr.io/cloud-builders/docker'
+  args: ['build', '-t', 'gcr.io/$PROJECT_ID/trainer_image_covertype_vertex:latest', '.']
+  dir: trainer_image_vertex
```

Comment:
don't we need to put _PIPELINE_FOLDER back here, so it is not `workspace/trainer_image_vertex` but `workspace/immersion/kubeflow_pipelines/pipelines/solutions/trainer_image_vertex`? I think my build fails because of this.

Reply:
this is resolved by another change in the yaml